External/FEXCore/Source/Interface/Core/Core.cpp

```diff
@@ -126,7 +126,7 @@ namespace DefaultFallbackCore {
 namespace FEXCore::Context {
   Context::Context()
     : FrontendDecoder {this}
-    , SyscallHandler {this} {
+    , SyscallHandler {FEXCore::CreateHandler(64, this)} {
```

Comment:
what does `64` mean here?

Reply:
You're right. Was difficult to tell what it was doing there.
Switched over to an enum to know if operating in 32bit or 64bit mode.
32bit mode not implemented yet.